src/Task/Development/loadTasks.php

```diff
@@ -36,7 +45,10 @@ protected function taskSemVer($pathToSemVer = '.semver')
      */
     protected function taskServer($port = 8000)
     {
-        return new PhpServer($port);
+        return $this->taskAssembler()->assemble(
```

Comment:
I don't see what's wrong about the formatting of this line. Looks the same as all of the others like it. No tabs that I can see. Odd.